README.adoc

```diff
@@ -87,8 +87,8 @@ settings, other beans, and various property settings.
 such as setting up a `DispatcherServlet`. Spring Boot adds it  automatically when it sees
 `spring-webmvc` on the classpath.
 - `@ComponentScan`: Tells Spring to look for other components, configurations, and
-services in the the `com.example.testingweb` package, letting it find the
-`HelloController` class.
+services in the package where your annotated `TestingWebApplication` class resides
+(`com.example.testingweb`), letting it find the `com.example.testingweb.HelloController` class.
```

Comment:
Please close the open parenthesis on this line. And you can drop ` class` at the end.

Reply:
Hi @Buzzardo, I have resolved it with this commit https://github.com/spring-guides/gs-testing-web/commit/70d438724b363c6e4a63c83a860843e715e2ea5b